### PR TITLE
Change conditions of node-exporter tasks execution

### PR DESCRIPTION
**Pull request by atsikham**
The second PR related to #2821 to avoid conflicts between different types of Node Exporter installation. In addition, not to create a few PRs, added a preflight check that kubeconfig exist for k8s_as_cloud_service: true in upgrade mode.

### CODE REVIEW

**Review by erzetpe — DISMISSED** [review]

**Review by erzetpe — DISMISSED** [review]

File: ansible/playbooks/roles/preflight/tasks/common/main.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  block:
+    - name: Check if kubeconfig file exists
+      stat:
+        path: "{{ kubeconfig.local }}"
+        get_attributes: false
+        get_checksum: false
+        get_mime: false
+      register: stat_kubeconfig
+
+    - assert:
```
Comment (przemyslavic) [inline ansible/playbooks/roles/preflight/tasks/common/main.yml]: Maybe worth adding a name for the task here?
```
2022-03-28T11:19:17.3490675Z [38;21m11:19:17 INFO cli.src.ansible.AnsibleCommand - TASK [preflight : assert] ******************************************************
2022-03-28T11:19:17.3829459Z [31;21m11:19:17 ERROR cli.src.ansible.AnsibleCommand - fatal: [ci-13nexpazubflannel-kubernetes-master-vm-0 -> localhost]: FAILED! => {"assertion": "stat_kubeconfig.stat.exists", "changed": false, "evaluated_to": false, "msg": "Kubeconfig file is not found. Copy/create it at /shared/build/13nexpazubflannel/kubeconfig."}
```

Comment (atsikham) [reply ansible/playbooks/roles/preflight/tasks/common/main.yml]: Done

**Review by przemyslavic — APPROVED** [review]

**Review by erzetpe — APPROVED** [review]

**Pull request merged**